WordPress/src/main/java/org/wordpress/android/util/image/ImagePlaceholderManager.kt

```diff
@@ -0,0 +1,26 @@
+package org.wordpress.android.util.image
+
+import org.wordpress.android.R
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class ImagePlaceholderManager @Inject constructor() {
+    fun getErrorImage(imgType: ImageType): Int? {
+        return when (imgType) {
+            ImageType.PHOTO -> R.color.grey_lighten_30
+            ImageType.VIDEO -> R.color.grey_lighten_30
+            ImageType.AVATAR -> R.drawable.ic_placeholder_gravatar_grey_lighten_20_100dp
+            ImageType.BLAVATAR -> R.drawable.ic_placeholder_blavatar_grey_lighten_20_40dp
+        }
+    }
+
+    fun getPlaceholderImage(imgType: ImageType): Int? {
```

Comment:
Maybe let's rename both to `getPlaceholderResource` since they are not really returning images?